src/policy/immix/immixspace.rs

```diff
@@ -591,7 +591,11 @@ impl<VM: VMBinding> ImmixSpace<VM> {
 
     fn unlog_object_if_needed(&self, object: ObjectReference) {
         if self.space_args.unlog_object_when_traced {
-            VM::VMObjectModel::GLOBAL_LOG_BIT_SPEC.mark_as_unlogged::<VM>(object, Ordering::SeqCst);
+            // Every immix line is 256 bytes, which is mapped to 4 bytes in the side metadata.
```

Comment:
A line is 256 bytes now, but I anticipate someone may want to do experiments by changing the line size to be extremely large or extremely small.  If it is too small, this optimisation will no longer be valid.  We can

-   Do an assertion that the line is sufficiently large, or
-   Use `if-else` statement to conditionally use this optimisation if the line is sufficiently large.